platform/security/certificate/security-certificate-generator/pom.xml

```diff
@@ -68,6 +68,16 @@
             <groupId>commons-io</groupId>
             <artifactId>commons-io</artifactId>
         </dependency>
+        <dependency>
+            <groupId>ddf.security.core</groupId>
+            <artifactId>security-core-api</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>*</groupId>
```

Comment:
Might want to comment on why the transitive dependencies need to be excluded here.